openstates/al/bills.py

```diff
@@ -50,30 +49,25 @@ class ALBillScraper(BillScraper):
     DATE_FORMAT = '%m/%d/%Y'
 
     def accept_response(self, response, **kwargs):
```

Comment:
add a comment about how this is a scrapelib function we're redefining for easier future maintainability?